netebpfext/sys/net_ebpf_ext_drv.c

```diff
@@ -178,3 +204,9 @@ DriverEntry(_In_ DRIVER_OBJECT* driver_object, _In_ UNICODE_STRING* registry_pat
     NET_EBPF_EXT_LOG_EXIT();
     return status;
 }
+
+_Ret_notnull_ DEVICE_OBJECT*
+ebpf_driver_get_device_object()
```

Comment:
nit The other static objects around it don't have getters - is it needed?
+not currently used, right?

Reply:
This was necessary as a result of pulling in the other dependencies. I could not compile without it.

Reply:
sounds good, mind adding this as a short note there?